packages/fxa-shared/README.md

```diff
@@ -100,7 +100,7 @@ require('fxa-shared/tracing/node-tracing').init(config.get('tracing'));
 ```
 
 To see traces on your local system, simply enable Jaeger exports on the service. This can be done by setting the environment
```

Comment:
nit: Probably say `otel` instead of Jaeger.

Generally, I wonder if we need to have two sections, one for browser and one for node?

Reply:
Updated accordingly...